src/CalcViewModel/GraphingCalculator/GraphingSettingsViewModel.h

```diff
@@ -244,6 +248,7 @@ namespace CalculatorApp::ViewModel
                 if (value && m_Graph != nullptr && m_Graph->TrigUnitMode != (int)Graphing::EvalTrigUnitMode::Degrees)
                 {
                     m_Graph->TrigUnitMode = (int)Graphing::EvalTrigUnitMode::Degrees;
+                    TraceLogger::GetInstance()->LogGraphSettingsChanged(GraphSettingsType::TrigUnits);
```

Comment:
I think Rudy’s comment for moving under the RaisePropertyChanged applies here and below